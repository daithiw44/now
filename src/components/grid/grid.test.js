import React from 'react';
import { shallow } from 'enzyme';
import { withStyles } from '@material-ui/core/styles';
import Grid from './grid';
describe('SharedButton Component', () => {

  let wrapper;
  beforeEach(() => {
    const props = {
      emails:[{
        avatar: "http://dummyimage.com/100x100.jpg/dddddd/000000",
        dtFormat: "26 February 2016, 10:05",
        email: "awilliams8@indiegogo.com",
        firstName: "Aaron",
        id: "555f4e1b-90a6-4079-850d-54001f29ce44",
        lastName: "Williams",
        message: "Quisque ut erat. Curabitur gravida nisi at nibh.",
        timestamp: "2016-02-26T10:05:17Z",
        userId: "c980934a-f54e-459e-8137-0ec3654cdafb"
      }]
    };
    wrapper = shallow(<Grid {...props} />);
  });

  it('renders', () => {
    expect(wrapper).not.toBeNull();
    expect(wrapper).toMatchSnapshot();
  });
})
