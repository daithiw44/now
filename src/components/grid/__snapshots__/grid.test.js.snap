// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SharedButton Component renders 1`] = `
<WithStyles(ForwardRef(ListItem))
  alignItems="flex-start"
  key="555f4e1b-90a6-4079-850d-54001f29ce44"
>
  <WithStyles(ForwardRef(Tooltip))
    aria-label="awilliams8@indiegogo.com"
    arrow={true}
    title="awilliams8@indiegogo.com"
  >
    <WithStyles(ForwardRef(ListItemAvatar))>
      <WithStyles(ForwardRef(Avatar))
        alt="Aaron Williams"
        src="http://dummyimage.com/100x100.jpg/dddddd/000000"
        title="Aaron Williams"
      />
    </WithStyles(ForwardRef(ListItemAvatar))>
  </WithStyles(ForwardRef(Tooltip))>
  <WithStyles(ForwardRef(ListItemText))
    primary="26 February 2016, 10:05"
    secondary={
      <React.Fragment>
        <ForwardRef(WithStyles)
          color="textPrimary"
          component="span"
          variant="body2"
        >
          Aaron
           
          Williams
           -
        </ForwardRef(WithStyles)>
        <span>
           
          Quisque ut erat. Curabitur gravida nisi at nibh.
           
        </span>
      </React.Fragment>
    }
  />
</WithStyles(ForwardRef(ListItem))>
`;
